### DISCUSSION TITLE
option for auto-install lsp

### DISCUSSION THREAD

**Discussion by matan-h** [helix-editor/helix#2086] [Ideas]
now, when the user want to use helix, he need to look at `How to install the default language servers` on the GitHub wiki,then use `hx --health` for check if lsp is installed,then install it,then maybe add it to $PATH.
maybe helix can prompt the user for auto-install the lsp of the file. (or point to the wiki)

like prompt:

_do you want to install recommended lsp for python ([python-lsp-server](https://github.com/python-lsp/python-lsp-server)) using pipx? [y/N]_

**Comment by sudormrfbin** [comment]
Pointing to the wiki seems like a good idea, but installing the server itself would be too big a burden maintenance wise. Maybe plugins in the future can solve this with "bundles" for languages.

**Reply by matan-h** [reply]
I don't know how to commit to my fork, without adding it to existing pr,so I delete it right after,but I made simple  println of wiki link in  checkhealth at a1bec0ac0a3b723898d0136b029125ed14655582

**Comment by David-Else** [comment]
https://github.com/williamboman/nvim-lsp-installer/ contains all the logic to install all the language servers, it could be converted to a bash script for universal use.
